whileLoop/aufgabe1.java

```diff
@@ -0,0 +1,31 @@
+package whileLoop;
+/*
+Aufgabe 1
+        B) Schreiben folgendes Snippet anhand einer While Schleife um:
+        for (int i = 0; i < 10; i++) {
+        if ((i%2) == 0) {
+        System.out.println(i + " ist eine gerade Zahl")
+        } else {
+        System.out.println(i + " ist eine ungerade Zahl");
+        }
+    }
+*/
+public class aufgabe1 {
```

Comment:
Class name shou;d start from the upper case and no numbers in it, e.g. `AufgabeEins`